### DISCUSSION TITLE
q-scroll-area cannot use height:100%

### DISCUSSION THREAD

**Discussion by lajxw** [quasarframework/quasar#12710] [General - Components / Directives / etc] [answered]
Is there any way to use height: 100% in q-scroll-area

**Comment by metalsadman** [comment]
see examples https://quasar.dev/layout/drawer#basic
using class `fit` https://quasar.dev/style/other-helper-classes#size-related

**Reply by lajxw** [reply]
After using fit, the height of q-scrollarea is still 0

```
<template>
  <q-page class="main-content">
    <q-scroll-area
      class="fit"
      ref="scrollArea"
      :thumb-style="thumbStyle"
      :visible="false"
    >
      <div v-for="n in 100" :key="n" class="q-py-xs">
        Lorem ipsum dolor sit amet, consectetur adipisicing
        elit, sed do eiusmod tempor incididunt ut labore et
        dolore magna aliqua.
      </div>
    </q-scroll-area>
  </q-page>
</template>
```

**Reply by metalsadman** [reply]
the parent must have a height, try wrapping it in a div with column class.

**Comment by metalsadman** [comment, marked as answer]
https://codepen.io/metalsadman/pen/YzEoWNa?editors=1010

**Reply by lajxw** [reply]
I want to use `q-scroll-area` in `q-page` instead of `q-drawer`.
I can't find a way to use q-scroll-area in q-page.

**Reply by metalsadman** [reply]
check the codepen above.

> https://codepen.io/metalsadman/pen/YzEoWNa?editors=1010
